src/main/java/Zjazd1/PolishFlag.java

```diff
@@ -0,0 +1,70 @@
+package Zjazd1;
+
+import java.util.Scanner;
+
+import static Zjazd1.Color.ANSI_BRIGHT_RED;
+import static Zjazd1.Color.ANSI_BG_RED;
+import static Zjazd1.Color.ANSI_RESET;
+import static Zjazd1.Color.ANSI_BG_BLACK;
+
+/**
+ * print Polish flag
+ * author: Michal Wadas
+ **/
+public class PolishFlag {
+
+    public static void main(String[] args) {
+        runProgram();
+    }
+
+    private static void runProgram() {
+        int size = 0;
+        Scanner scanner = new Scanner(System.in);
+        System.out.println("Choose size of the flag: 1= small , 2= medium , 3=large");
+
+        int flagSize = scanner.nextInt();
+        if (flagSize == 1) {
+            size = 2;
+        } else if (flagSize == 2) {
+            size = 4;
+        } else if (flagSize == 3) {
+            size = 6;
+        } else {
+            System.out.println("Bad input!");
+        }
+        System.out.println();
+        japaneseFlag(size);
+    }
+
+    private static void japaneseFlag(int size) {
```

Comment:
avoid function returning void, this function should be refactored to return a string rather than be mutating external-device (stdout in this case).